docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
```

Comment:
It's not required, but it does make running the artisan command easier to run since you can simply run against the `firefly-iii` container. If there are other ways to get that done though, I don't see a reason to leave this in.